CMakeLists.txt

```diff
@@ -7,11 +7,13 @@ set(LIMEREPORT_VERSION_RELEASE 6)
 
 option(ENABLE_ZINT "Enable libzint build for barcode support" OFF)
 option(LIMEREPORT_STATIC "Build LimeReport as static library" OFF)
+option(USE_QT6 "Use Qt6" OFF)
 
-find_package(
-  QT NAMES Qt6 Qt5
-  COMPONENTS Core Widgets Sql Network Xml Svg Qml PrintSupport REQUIRED
-  )
+if(USE_QT6)
+    find_package(QT NAMES Qt6)
+else()
```

Comment:
What does that fix?

Reply:
This can help with choosing what Qt version Limereport is gonna be built with ( Qt5 or Qt6 ) in the case when both of qt versions are installed.
I had that problem when I was building Limereport on my linux machine and also when cross building for windows with mingw.